addons/portal/views/portal_templates.xml

```diff
@@ -720,7 +720,7 @@
                 <img class="rounded object-fit-cover" t-att-src="image_data_uri(user_id.partner_id.avatar_128)" alt="Contact" width="50"/>
                 <div class="d-flex flex-column justify-content-center">
                     <h5 class="mb-0" t-out="user_id.name"/>
-                    <p class="mb-0 text-muted" t-out="user_id.company_name"/>
+                    <p class="mb-0 text-muted" t-out="user_id.parent_id.parent_name"/>
```

Comment:
Isn't it one extra level ? Aka either parent_id.name, either parent_name, but seems here we get two levels up ?